jiant/utils/string_comparing.py

```diff
@@ -0,0 +1,47 @@
+import re
+import string
+import collections
+
```

Comment:
Note to other reviewers: I confirmed that the functions in this file are identical (except for comments and some very minor syntax differences) to the official evaluation script [provided by ReCoRD's authors](https://sheng-z.github.io/ReCoRD-explorer/evaluation.py).